modules/ossm-automatic-gateway-injection.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-traffic-manage.adoc
+
+:_content-type: CONCEPT
+[id="ossm-automatic-gateway-injection_{context}"]
+= Enabling gateway injection
+
+Gateway configurations are applied to standalone Envoy proxies that are running at the edge of the mesh, rather than sidecar Envoy proxies running alongside your service workloads. Because gateways are Envoy proxies, you can configure {SMProductShortName} to automatically inject gateways similar to how you can automatically inject sidecars.
```

Comment:
```suggestion
Gateway configurations are applied to standalone Envoy proxies running at the edge of the mesh, rather than sidecar Envoy proxies running alongside your service workloads. Because gateways are Envoy proxies, you can configure {SMProductShortName} to automatically inject gateways similar to how you can automatically inject sidecars.
```
- consider omitting to tighten phrasing and make parallel with construction in second half of senteence